content/_data/changelogs/weekly.yml

```diff
@@ -1510,6 +1510,30 @@
       pull: 3123
     # pull: 3131 too minor (French translation fix)
     # pull: 3132 too minor (Javadoc)
+- version: "2.91"
+  date: 2017-11-19
+  changes:
+    - type: major rfe
+      references:
+        - issue: 36088
+        - issue: 39179
+      pull: 3133
+      message: >
+        Use Java NIO library instead of native code to create and detect symbolic links and Windows junctions to improve compatibility and robustness.
+    - type: bug
+      issue: 34138
+      pull: 3042
+      message: >
+        Prevent concurrent installation of Maven on the same node to prevent problems.
+    - type: rfe
+      issue: 47718
+      pull: 3114
+      message: >
+        Developer: Deprecate the ambiguous <code>User#getUser(String)</code> in favor of the new <code>User#getOrCreate(String)</code>.
```

Comment:
Sorry, I forgot to modify the changelog entry, "of the User#getById() or the new Use#getOrCreateByIdOrFullName() methods" or so